lib/date_time/date_time.c

```diff
@@ -712,8 +716,10 @@ int date_time_update_async(date_time_evt_handler_t evt_handler)
 
 int date_time_clear(void)
 {
-	time_aux.date_time_utc = 0;
-	time_aux.last_date_time_update = 0;
+	/* TODO: I guess we shouldn't set Zephyr time to year 1970 again?
+	 * Do we need the entire function? Doesn't seem to be used in nrf repo.
```

Comment:
@simensrostad Can you comment on what to do with this function?

Reply:
Agree. It doesn't really provide much. It can probably be removed.

Reply:
`date_time_clear()` has been removed now. I need to update documentation and tests.

Reply:
We need date_time_clear in tests, not really anywhere else so it's been restored.